### DISCUSSION TITLE
Russian translation

### DISCUSSION THREAD

**Discussion by cryptozoy** [OpenHantek/OpenHantek6022#164] [Recurring tasks]
<location filename="../src/mainwindow.cpp" line="80"/>
        <source>Demo Mode</source>
        <translation>демонстрационного режима</translation> Change to <translation>Демо Режим</translation>

        <location filename="../src/selectdevice/selectsupporteddevice.cpp" line="73"/>
        <source>&lt;p&gt;Please make sure you have installed the windows usb driver correctly&lt;/p&gt;</source>
        <translation>&lt;p&gt;Убедитесь в корректно установленных USB драйверах для Windows&lt;/p&gt;</translation> Change to <translation>&lt;p&gt;Убедитесь в корректно установленных USB драйверах для Windows.&lt;/p&gt;</translation>

        <location filename="../src/selectdevice/selectsupporteddevice.cpp" line="86"/>
        <source>&lt;hr/&gt;&lt;p&gt;Even without a device you can explore the program&apos;s function. Just press the &lt;b&gt;Demo Mode&lt;/b&gt; button below.&lt;/p&gt;</source>
        <translation>&lt;hr/&gt;&lt;p&gt;Вы можете изучить функции программы без устройства, просто нажмите кнопку &lt;b&gt;демонстрационного режима&lt;/b&gt; кнопки.&lt;/p&gt;</translation> Change to <translation>&lt;hr/&gt;&lt;p&gt;Вы можете изучить функции программы без устройства, просто нажмите кнопку &lt;b&gt;Демо Режима&lt;/b&gt;.&lt;/p&gt;</translation>

**Comment by cryptozoy** [comment]
<location filename="../src/main.cpp" line="81"/>
        <source>Demo mode without scope HW</source>
        <translation>Демонстрационный режим без аппаратного обеспечения</translation> to <translation>Демо режим без аппаратуры осциллографа</translation>

        <location filename="../src/selectdevice/selectsupporteddevice.ui" line="75"/>
        <source>Demo Mode</source>
        <translation>демонстрационного режима</translation> to <translation>Демо Режим</translation>

        <location filename="../src/selectdevice/deviceslistmodel.cpp" line="24"/>
        <source>Status</source>
        <translation>Status</translation> to <translation>Статус</translation>

        <location filename="../src/configdialog/DsoConfigFilePage.cpp" line="21"/>
        <source>Take a screenshot of the display area only</source>
        <translation>Сделайте снимок экрана только в области отображения</translation> to <translation>Делать снимок экрана только в области отображения</translation>

**Comment by Ho-Ro** [comment]
Let's keep this issue open for future translation.

**Comment by cryptozoy** [comment]
OK. Let it be. )))

**Comment by Ho-Ro** [comment]
Hi, please check:

        <source>Set slowest possible timebase&lt;br/&gt;(&lt;b&gt;GUI may become very unresponsible!&lt;/b&gt;)</source>
        <translation>Установите самую медленную временную базу&lt;br/&gt;(&lt;b&gt;GUI может стать очень медленным!&lt;/b&gt;)</translation>

and

        <source>Minimal time between captured frames&lt;br/&gt;(Longer times reduce the CPU load)</source>
        <translation>Минимальное время между захваченными кадрами&lt;br/&gt;(Дольше времени снижает нагрузку на процессор)</translation>

**Comment by cryptozoy** [comment]
```
<source>Set slowest possible timebase&lt;br/&gt;(&lt;b&gt;GUI may become very unresponsible!&lt;/b&gt;)</source>
<translation>Установите самую медленную временную базу&lt;br/&gt;(&lt;b&gt;GUI может стать очень медленным!&lt;/b&gt;)</translation>
```
```
<translation>Установите максимальное Время/Деление&lt;br/&gt;(&lt;b&gt;GUI может начать подвисать!&lt;/b&gt;)</translation>
```
and
```
<source>Minimal time between captured frames&lt;br/&gt;(Longer times reduce the CPU load)</source>
<translation>Минимальное время между захваченными кадрами&lt;br/&gt;(Дольше времени снижает нагрузку на процессор)</translation>
```
```
<translation>Минимальное время между захватами кадров&lt;br/&gt;(Больше времени - ниже нагрузка на процессор)</translation>
```

**Comment by cryptozoy** [comment]
<location filename="../src/configdialog/DsoConfigScopePage.cpp" line="77"/>
        <source>Save settings on exit</source>
        <translation>Сохранять настройки при выходе</translation>

        <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="81"/>
        <source>Save settings now</source>
        <translation>Сохранить текущие настройки</translation>

**Comment by Ho-Ro** [comment]
@spam-receiver:  New text, please provide a good translation:

    <message>
        <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="83"/>
        <source>Scope has hardware modification for AC coupling (restart needed to apply the change)</source>
        <translation type="unfinished"></translation>
    </message>

**Comment by cryptozoy** [comment]
```
<message>
    <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="83"/>
    <source>Scope has hardware modification for AC coupling (restart needed to apply the change)</source>
    <translation>Осциллограф имеет аппаратную модификацию под переменный ток (для активации требуется перезапуск)</translation>
</message>
```

**Comment by cryptozoy** [comment]
[openhantek_ru.ts.gz](https://github.com/OpenHantek/OpenHantek6022/files/4830704/openhantek_ru.ts.gz)

Everything looks nice in Fedora 32.

**Comment by cryptozoy** [comment]
[openhantek_ru.ts.gz](https://github.com/OpenHantek/OpenHantek6022/files/4842745/openhantek_ru.ts.gz)

Make work a hot key for checkbox "SP2" ("Спектр 2"). It was in conflict with menu "Help" ("Помощь").

**Comment by cryptozoy** [comment]
[openhantek_ru.ts.zip](https://github.com/OpenHantek/OpenHantek6022/files/5484839/openhantek_ru.ts.zip)

Shortened the names in the math selection dropdown list ("К" instead of "Канал") to reduce the width of the "Voltage" dock in Russian localization after https://github.com/OpenHantek/OpenHantek6022/commit/d845b2e0bf99dbb9dceab84d127a00410a8b6d64

**Comment by cryptozoy** [comment]
Offtopic:

Spelling mistake in README.md - one letter is missing. "Continous Integration" has to be "Continuous Integration".

**Reply by Ho-Ro** [reply]
thx, fixed with bb3c550

**Comment by cryptozoy** [comment]
More:

Voltage and Spectrum view for all device supported **chanels** (channels).

The *.tar.gz **achives** (archives) contain the same files as the *.deb and *.rpm packages for quick testing - do not use for a permanent **intallation** (installation). Do not report any issues about the *.tar.gz!

**Comment by cryptozoy** [comment]
[openhantek_ru.zip](https://github.com/OpenHantek/OpenHantek6022/files/8265418/openhantek_ru.zip)

**Comment by cryptozoy** [comment]
[openhantek_ru.zip](https://github.com/OpenHantek/OpenHantek6022/files/8319568/openhantek_ru.zip)

**Comment by cryptozoy** [comment]
[openhantek_ru.zip](https://github.com/OpenHantek/OpenHantek6022/files/8406211/openhantek_ru.zip)

**Comment by cryptozoy** [comment]
[openhantek_ru.zip](https://github.com/OpenHantek/OpenHantek6022/files/8420098/openhantek_ru.zip)

**Comment by cryptozoy** [comment]
[openhantek_ru.zip](https://github.com/OpenHantek/OpenHantek6022/files/8473899/openhantek_ru.zip)

**Comment by cryptozoy** [comment]
[openhantek_ru.zip](https://github.com/OpenHantek/OpenHantek6022/files/8516724/openhantek_ru.zip)

**Comment by cryptozoy** [comment]
Zoom area height factor -> Коэффициент высоты области увеличения
[openhantek_ru.zip](https://github.com/OpenHantek/OpenHantek6022/files/8886521/openhantek_ru.zip)

**Comment by cryptozoy** [comment]
"OpenHantek.desktop" localized with russian generic name and comment. I propose to do the same for all other languages.
[OpenHantek.zip](https://github.com/OpenHantek/OpenHantek6022/files/9121964/OpenHantek.zip)
